cmd/daemon/command/start.go

```diff
@@ -93,5 +94,7 @@ func StartDaemon() *cobra.Command {
 	//nolint:errcheck
 	command.MarkFlagRequired("environment")
 	logConfiguration = log.RegisterFlags(command)
+
+	flux_notification_controller.StartHttpServer()
```

Comment:
This needs to go into the `RunE` function above along with the rest of the "kubernetes watcher" stuff. That way it is started in the lifetime of the general application 😄  So just before the `Deamon started` log line maybe.